generated-examples/todo-with-wallet/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TODO List | Web5.js App</title>
+    <link href="https://cdn.jsdelivr.net/gh/TBD54566975/web5-omg@main/styles.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <h1>TODO List</h1>
+    <ul id="todo-list">
+    </ul>
+
+
+    <button type="button" id="connect_button">DID Connect</button>
+    <div>
+      <span>Security Code: </span><strong id="security_code"></strong>
+    </div>
+  
+
+    <form id="todo-form">
+      <input type="text" id="todo-input" placeholder="Enter a new todo">
+      <button type="submit" id="add-todo-btn">Add Todo</button>
+    </form>
+
+    
+
+    <script type="module">
+
+      /* boilerplate start - required for every web5 app */
+      import { Web5 } from 'https://cdn.jsdelivr.net/npm/@tbd54566975/web5@0.4.0-unstable.e6470ce-2023.4.01-19-39-11/dist/browser.mjs';
+
+      const web5 = new Web5();
+      let myDid = '';
+
+      web5.addEventListener('open', (event) => {
+        document.querySelector('#security_code').textContent = event.detail.pin;
+      });
+      web5.addEventListener('connection', (event) => {
+        myDid = event.detail.did;
+        alert('Connection succeeded!');
+        loadTodos();
+
+        // Load todos on page load
+        window.addEventListener('load', loadTodos);
+
+      });
+      web5.addEventListener('close', (event) => {
+        alert('Connection was denied');
+      });
+      web5.addEventListener('error', (event) => {
+        console.error(event);
+      });
+
+      document.querySelector('#connect_button').addEventListener('click', async event => {
+        event.preventDefault();
+        web5.connect();
+      });
+
+      function base64UrlToString(encodedData) {
+        return web5.dwn.SDK.Encoder.bytesToString(web5.dwn.SDK.Encoder.base64UrlToBytes(encodedData));
+      }
+      function base64UrlToObject(encodedData) {
+        return web5.dwn.SDK.Encoder.base64UrlToObject(encodedData);
+      }
+      /* boilerplate end */
+
+      const todoList = document.getElementById('todo-list');
+      const todoForm = document.getElementById('todo-form');
+      const todoInput = document.getElementById('todo-input');
+
+
+      async function loadTodos() {
+        console.log("loading dids for", myDid);
+        todoList.innerHTML = '';
+        const response = await web5.dwn.records.query(myDid, {
+          author: myDid,
+          message: {
+            filter: {
+              schema: 'todo',
+            }
+          }
+        });
+        let j = await response.json();
```

Comment:
hey @frankhinek this is a bit weird - I noticed it is needed when it is using the wallet (when its a cors Response). Is this to be expected?

Reply:
(but it works when you do this)